js/client-battle-tooltips.js

```diff
@@ -366,14 +366,16 @@ var BattleTooltips = (function () {
 			var item = '';
```

Comment:
Ah, so I could have just changed this to `'None'` (plus associated code cleanup) in the first place...